### ISSUE TITLE
Destroying a Silo destroys the game

### ISSUE THREAD

**Issue by twothe** [BluSunrize/ImmersiveEngineering#937]
If a full Silo is destroyed and the items drop on the ground, the game tries to render approximately 5000 items and dies.

Since the ticks/s also drop massively, there are basically only two things that can be done: wait approximately 5-6 hours till the "5 minutes" despawn time is reached, or start a new world/load a backup.

**Comment by laz2727** [comment]
It isn't just about rendering. Stacking them together also takes a lot.
To op: mcedit should help.

**Comment by malte0811** [comment]
I just destroyed a silo containing 10k Hoppers and it dropped no more than 1 stack of 64 hoppers, which should not be a problem. (Tested with a build of commit b261342d77a62148a256a8524a02317875e065fc (latest commit of my current PR))
@twothe What other mods do you have installed and what version of IE are you using?

**Comment by laz2727** [comment]
That sounds like a separate issue.

**Comment by malte0811** [comment]
It is not an issue, it is the intended behavior for all I know. It would of course be possible to make it not drop anything at all, but that would cause an unnecessary mess in the code.

**Comment by twothe** [comment]
The issue is a bit tough to pinpoint, as the world in question causes the game to freeze forever as soon as you join. The only thing I could find so far were >5000 stacks of seeds beneath the seed Silo.

**Comment by theunkn0wn1** [comment]
Also unable to reproduce when testing against Ie build 8b379db3dd4d22bbd14a406755c902646e6bed3f

--------------
I filled a silo with as much Vanilla seed as it could hold and when destroyed it only dropped two stacks of seed onto the ground.
Also: The Silo does not have anywhere near enough storage capacity to store 5k **stacks** of seed, barely enough to store 41k seed. This value was identified experimentally using the setup shown below
![image](https://cloud.githubusercontent.com/assets/3110986/12769232/eff989c0-c9ca-11e5-8e49-cb25601d7fa0.png)

-----------------------
Regardless of what gamemode the user was in, survival or Creative, the silo only dropped a maximum of two stacks of Seed.

@twothe What version of IE are you using? Can we please get a copy of the Logs? Is this issue reproducible?

**Comment by twothe** [comment]
I just spent an hour to reproduce the issue and learned that this is most likely caused by ProjectRed Pipes and not by the Silo itself (or in combination).

The seeds were pulled out from underneath but never delivered, causing seeds to accumulate and pile up to an insane amount. But since the seeds were directly beneath the silo (and not inside a pipe) I was under the impression this had been caused by the Silo itself. Removing ProjectRed caused the seeds to despawn pretty fast, and I could see that the Silo was in fact intact and the broken display I had before obviously a result of the client not being able to render anymore.
